clients/vscode-hlasmplugin/syntaxes/hlasmListing.tmLanguage.json

```
@@ -0,0 +1,397 @@
+{
+  "$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+  "name": "IBM HLASM Listing",
+  "scopeName": "source.hlasmListing",
+  "patterns": [
+    {
+      "begin": "  Loc    Object Code      Addr1    Addr2    Stmt  Source Statement",
+      "end": "                                                 Relocation Dictionary",
+      "patterns": [ {"include": "#lines_to_ignore"}, {"include" : "#listing_syntax_long"}]
+    },
+    {
+      "begin": "  Loc  Object Code    Addr1 Addr2  Stmt   Source Statement",
+      "end": "                                                 Relocation Dictionary",
+      "patterns": [ {"include": "#lines_to_ignore"}, {"include" : "#listing_syntax"}]
+    },
+    {
+      "begin": "(.)  Loc    Object Code      Addr1    Addr2    Stmt  Source Statement",
+      "end": "(.)                                                 Relocation Dictionary",
+      "patterns": [ {"include": "#lines_to_ignore_endevor"}, {"include" : "#listing_syntax_endevor_long"}]
+    },
+    {
+      "begin": "(.)  Loc  Object Code    Addr1 Addr2  Stmt   Source Statement",
+      "end": "(.)                                                 Relocation Dictionary",
+      "patterns": [ {"include": "#lines_to_ignore_endevor"}, {"include" : "#listing_syntax_endevor"}]
+    }
```

It looks like Endevor places the member name at the beginning of the line, so let's relax the end condition a bit
```suggestion
    {
      "begin": "(.)  Loc    Object Code      Addr1    Addr2    Stmt  Source Statement",
      "end": "(1.{8})                                         Relocation Dictionary",
      "patterns": [ {"include": "#lines_to_ignore_endevor"}, {"include" : "#listing_syntax_endevor_long"}]
    },
    {
      "begin": "(.)  Loc  Object Code    Addr1 Addr2  Stmt   Source Statement",
      "end": "(1.{8})                                         Relocation Dictionary",
      "patterns": [ {"include": "#lines_to_ignore_endevor"}, {"include" : "#listing_syntax_endevor"}]
    }
```